src/renderer/wg_engine/tvgWgGeometry.h

```diff
@@ -49,34 +49,62 @@ class WgPoint
     WgPoint operator / (const float c) const { return { x / c, y / c }; }
 
     WgPoint negative() const { return {-x, -y}; }
-    void negate() { x = -x; y = -y; }
-    float length() const { return sqrt(x*x + y*y); }
-
-    float dot(const WgPoint& p) const { return x * p.x + y * p.y; }
-    float dist(const WgPoint& p) const
-    { 
-        return sqrt(
-            (p.x - x)*(p.x - x) + 
-            (p.y - y)*(p.y - y)
-        ); 
-    }
-
-    void normalize()
-    {
-        float rlen = 1.0f / length();
-        x *= rlen;
-        y *= rlen;
-    }
-
-    WgPoint normal() const
-    {
-        float rlen = 1.0f / length();
-        return { x * rlen, y * rlen };
-    }
+    inline void negate() { x = -x; y = -y; }
+    inline float length() const { return sqrt(x*x + y*y); }
+    inline float dot(const WgPoint& p) const { return x * p.x + y * p.y; }
+    inline float dist(const WgPoint& p) const { return sqrt(dist2(p)); }
+    inline float dist2(const WgPoint& p) const { return ((p.x - x)*(p.x - x) + (p.y - y)*(p.y - y)); }
+    inline void normalize() { float rlen = 1.0f / length(); x *= rlen; y *= rlen; }
+    inline WgPoint normal() const { float rlen = 1.0f / length(); return { x * rlen, y * rlen }; }
+    inline WgPoint lerp(const WgPoint& p, float t) const { return { x + (p.x - x) * t, y + (p.y - y) * t }; };
 };
 
+
+struct WgMath
+{
+    Array<float> sinus;
+    Array<float> cosin;
+    bool initialized{};
+
+    void initialize();
+    void release();
+};
+
+
+struct WgPolyline
+{
+    // polyline points
```

Comment:
Suggestions:
1. These comments in the WgPolyline structure merely duplicate the variable names and seem unnecessary. Comments should have any hidden purpose or intention more than it.
2. Please append comments on one line if necessary.